### DISCUSSION TITLE
T4 template generator issue for Guid newid()

### DISCUSSION THREAD

**Discussion by sgashua** [linq2db/linq2db#3105] [Q&A] [answered]
I have table that contains guid using newid(). After I generated entities automatically, guid shows null after inserting/updating data. I have to add Column(SkipOnUpdate = true, SkipOnInsert = true) in every guid property of entities. 

Can fix it to add "Column(SkipOnUpdate = true, SkipOnInsert = true)" for guid with newid() during T4 generating?

Thanks.

**Comment by MaceWindu** [comment, marked as answer]
Check last example here https://linq2db.github.io/articles/T4.html#example-of-generation-process-customization
Column structure defined here https://linq2db.github.io/articles/T4.html#useful-members-and-data-structures
```cs
    public bool      SkipOnUpdate;
    public bool      SkipOnInsert;
```
